styles/signals.mapcss

```diff
@@ -747,6 +748,46 @@ node|z17-[railway=signal]["railway:signal:direction"]["railway:signal:minor"="AT
 	allow-overlap: true;
 }
 
+/********************************************/
+/* DE signals for helper engines type Ts    */
+/* DE Nachschieben einstellen Ts 1          */
+/********************************************/
+node|z14-[railway=signal]["railway:signal:direction"]["railway:signal:helper_engine"="DE-ESO:ts"]["railway:signal:helper_engine:form"=sign]["railway:signal:helper_engine:states"="DE-ESO:ts1"],
```

Comment:
Please remove the comma.

Reply:
Should be solved. And spotted a missing linebreak / wrong styling for the leading comment.